langroid/agent/chat_agent.py

```diff
@@ -730,6 +1119,10 @@ async def llm_response_async(
             if isinstance(message, str)
             else message.metadata.tool_ids if message is not None else []
         )
+
+        # If using strict output format, parse the output JSON
+        self._load_output_format(response)
```

Comment:
same comment as in sync case - since the `llm_response_messages_async` is doing it already, do we need it here?